cfme/services/requests.py

```diff
@@ -247,7 +249,9 @@ def in_requests(self):
 
 
 class RequestsView(RequestBasicView):
-    table = Table(locator='//*[@id="list_grid"]/table')
+    table = Table(locator=VersionPick({
+        Version.lowest(): '//*[@id="list_grid"]/table',
```

Comment:
Locator can be combined to //div[@id="gtl_div"]//table, to omit versioning

Reply:
@rlbabyuk I'll put your suggestion into separate PR. thanks for noticing this hint!